src/coreclr/interpreter/compiler.cpp

```diff
@@ -3791,7 +3791,7 @@ void InterpCompiler::GenerateCode(CORINFO_METHOD_INFO* methodInfo)
                     EmitConv(m_pStackPointer - 1, StackTypeI8, INTOP_CONV_OVF_U8_R8);
                     break;
                 case StackTypeI4:
-                    EmitConv(m_pStackPointer - 1, StackTypeI8, INTOP_CONV_I8_U4);
+                    EmitConv(m_pStackPointer - 1, StackTypeI8, INTOP_CONV_OVF_U8_I4);
```

Comment:
I think the original was ok. The U4 always fits into I8, so it seems we don't need to have the special OVF opcode for this one.

Reply:
The source is always signed, and for the _UN variant we treat the signed source as if it's unsigned. If I understand everything correctly.

Reply:
I agree with @kg here. Looks like the original code was implementing by mystake the `CEE_CONV_OVF_U8_UN` instead.